src/app/client/src/app/modules/report/components/all-evidence/all-evidence.component.scss

```diff
@@ -46,9 +46,12 @@
 }
 .sbt-report-btns-panel-new {
   border-radius: 0px !important;
-  background: #e9e8d9 !important;
+ background-color: #fff;
```

Comment:
please use cooler variable we will not use the color like hardcoding for white we have in the _variable.scss